README.md

```diff
@@ -21,33 +21,34 @@ A simple Hello world page.
 
```

Comment:
- [ ] [ OPTIONAL ]
This live demo link is broken, Kindly Select the feature branch in GitHub pages for the live server. 
Your README is the entry point of your users and having it well written without any broken links gives a good first impression and maximizes the potential of your product to your users.